src/qml/QFieldSettings.qml

```diff
@@ -265,6 +256,54 @@ Page {
                       }
                   }
 
+                  GridLayout {
+                      Layout.fillWidth: true
+                      Layout.leftMargin: 20
+                      Layout.rightMargin: 20
+                      Layout.topMargin: 5
+                      Layout.bottomMargin: 40
+
+                      columns: 1
+                      columnSpacing: 0
+                      rowSpacing: 5
+
+                      visible: platformUtilities.capabilities & PlatformUtilities.AdjustBrightness
+
+                      Label {
+                          Layout.fillWidth: true
+                          text: qsTr('Dim screen when idling')
+                          font: Theme.defaultFont
+
+                          wrapMode: Text.WordWrap
+                      }
+
+                      Label {
```

Comment:
If we'd want to keep this in line with other elements, we need to put the gray description / helper to sit below the slider.